bin/keycloak-httpd-client-install

```diff
@@ -1218,39 +1289,54 @@ def main():
                                           keep_trailing_newline=True,
                                           undefined=jinja2.StrictUndefined,
                                           loader=jinja2.FileSystemLoader(
-                                              options.template_dir))
+                                              'templates'))
```

Comment:
Why was this changed from options.template_dir?

Reply:
This change was made to make the tool work for my specifically. I forgot to change it back before commiting, sorry! I will change it back!